packages/frontend/app/templates/assign-students.hbs

```diff
@@ -1,36 +1,9 @@
 {{page-title (t "general.admin")}}
 <BackToAdminDashboard />
-<section class="assign-students">
-  <div class="filters">
-    <div class="schoolsfilter" data-test-school-filter>
-      <FaIcon @icon="building-columns" @fixedWidth={{true}} />
-      {{#if this.hasMoreThanOneSchool}}
-        <select aria-label={{t "general.filterBySchool"}} {{on "change" (pick "target.value" (set this.schoolId))}}>
-          {{#each (sort-by "title" @model.schools) as |school|}}
-            <option
-              selected={{eq school this.selectedSchool}}
-              value={{school.id}}
-            >
-              {{school.title}}
-            </option>
-          {{/each}}
-        </select>
-      {{else}}
-        {{this.selectedSchool.title}}
-      {{/if}}
-    </div>
-    <div class="titlefilter" data-test-title-filter>
-      <input
-        aria-label={{t "general.filterByTitle"}}
-        placeholder={{t "general.pendingUserUpdates.filterBy"}}
-        type="text"
-        value={{this.query}}
-        {{on "input" (pick "target.value" (perform this.setQuery))}}
-      >
-    </div>
-  </div>
-  <AssignStudents
-    @school={{this.selectedSchool}}
-    @students={{this.filteredUnassignedStudents}}
-  />
-</section>
+<AssignStudents::Root
+  @model={{this.model}}
+  @query={{this.query}}
+  @schoolId={{this.schoolId}}
+  @setSchoolId={{this.setSchoolId}}
```

Comment:
I'm fine with this as is, but I also wouldn't mind ditching the actions in the controller and using `@setSchoolId={{set this.schoolId}}` here and below on set query.

Reply:
good point. i changed it. :heavy_check_mark: